test/python/algorithms/time_evolvers/test_trotter_qrte.py

```diff
@@ -88,16 +88,70 @@ def test_trotter_qrte_trotter_single_qubit_aux_ops(self):
 
         aux_ops_result = evolution_result.aux_ops_evaluated
         expected_aux_ops_result = [
-            (0.078073, {"variance": 0, "shots": 0}),
-            (0.268286, {"variance": 0, "shots": 0}),
+            [
+                (0, {"variance": 0, "shots": 0}),
+                (0, {"variance": 0, "shots": 0}),
+            ],
+            [
+                (0.078073, {"variance": 0, "shots": 0}),
+                (0.268286, {"variance": 0, "shots": 0}),
+            ]
         ]
 
-        means = [element[0] for element in aux_ops_result]
-        expected_means = [element[0] for element in expected_aux_ops_result]
+        means = [element[0][0] for element in aux_ops_result]
+        expected_means = [element[0][0] for element in expected_aux_ops_result]
```

Comment:
This connects to a comment above of calling the observables evaluated at each timestep `observables`. We should keep the old functionality of the `aux_ops_evaluated` as is for now to remain backward compatible 🙂